config/default.yml

```diff
@@ -169,6 +169,17 @@ Rails/Date:
     - strict
     - flexible
 
+Rails/DateAndTimeColumnName:
+  Description: >-
+                Enforces date/time columns naming conventions.
+                Specifically, `date` column should end with `_on` suffix,
+                `datetime` and `timestamp` - with `_at` suffix,
+                `time` - with `_time` suffix.
+  Enabled: pending
+  VersionAdded: '2.8'
+  Include:
+    - db/migrate/**/**.rb
```

Comment:
Multiple db setup reads `migrations_paths:` from `config/database.yml`. I'm not sure if there's a convention on how to place them. Mine are usually in `db/secondary_migrate/*.rb`.

Reply:
I changed it to
```
  Include:
    - db/**/*.rb
```

like in https://github.com/rubocop/rubocop-rails/pull/277
That way it will check all migration files (I hope no one places migration files using `migrations_paths:` into other folders than `db/` 😄 )